### ISSUE TITLE
ModuleNotFoundError: No module named 'click._bashcomplete'

### ISSUE THREAD

**Issue by andersy005** [NCAR/jupyter-forward#91]
The latest version of `click: 8.0.1` breaks `typer` which is one of our dependencies (see https://github.com/tiangolo/typer/issues/278)

```zsh
(playground) ➜  ~ jupyter-forward --help
Traceback (most recent call last):
  File "/Users/abanihi/.mambaforge/envs/playground/bin/jupyter-forward", line 6, in <module>
    from jupyter_forward.cli import main
  File "/Users/abanihi/.mambaforge/envs/playground/lib/python3.9/site-packages/jupyter_forward/cli.py", line 4, in <module>
    import typer
  File "/Users/abanihi/.mambaforge/envs/playground/lib/python3.9/site-packages/typer/__init__.py", line 31, in <module>
    from .main import Typer, run
  File "/Users/abanihi/.mambaforge/envs/playground/lib/python3.9/site-packages/typer/main.py", line 11, in <module>
    from .completion import get_completion_inspect_parameters
  File "/Users/abanihi/.mambaforge/envs/playground/lib/python3.9/site-packages/typer/completion.py", line 10, in <module>
    import click._bashcomplete
ModuleNotFoundError: No module named 'click._bashcomplete'
```

The current workaround is to downgrade `click` to an earlier version

```zsh
mamba install 'click<8'
```

**Comment by tiangolo** [comment]
I just released Typer `0.4.0`, which is compatible with **both** Click 7 and Click 8. :tada: 

Upgrading the dependency of Typer should handle the problem.

**Comment by andersy005** [comment]
Thank you for the heads-up, Sebastián!
